subprojects/docs/src/samples/userguide/dependencyManagement/definingUsingConfigurations/custom/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+// tag::custom-configuration[]
+val jasper by configurations.creating
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    jasper("org.apache.tomcat.embed:tomcat-embed-jasper:9.0.2")
+}
+
+task("preCompileJsps") {
+    doLast {
+        ant.withGroovyBuilder {
+            "taskdef"(mapOf("classname" to "org.apache.jasper.JspC",
+                            "name" to "jasper",
+                            "classpath" to jasper.asPath))
+            "jasper"(mapOf("validateXml" to false,
+                           "uriroot" to file("src/main/webapp"),
+                           "outputDir" to file("$buildDir/compiled-jsps")))
```

Comment:
:x: this can be simplified to:
```kotlin
            "taskdef"("classname" to "org.apache.jasper.JspC",
                            "name" to "jasper",
                            "classpath" to jasper.asPath)
            "jasper"("validateXml" to false,
                           "uriroot" to file("src/main/webapp"),
                           "outputDir" to file("$buildDir/compiled-jsps"))
```